suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -223,6 +223,9 @@ export const getBip43Type = (path: string) => {
     }
 };
 
+export const substituteBip43Path = (bip43Path: Bip43Path, accountIndex: string | number = '0') =>
```

Comment:
5b7d207d46ef6cbf6e29ceb179ef451acbf653b3 that was a little bit more work, as I had to dissect the current usages of `Bip43Path`, to determine at each place if we are dealing with template, final path, or both (in case of `getAccountTypeName`)